### DISCUSSION TITLE
Sobre guardias tradicionales para evitar multi importación de un mismo archivo

### DISCUSSION THREAD

**Discussion by notyourlasagna** [IIC2133-PUC/2024-1#36] [Tarea 0] [answered]
Para esta tarea, asumimos que los compiladores que utilizarán para corregir soportan #pragma once como vimos en la ayudantía verdad? que sino vi que las guardias tradicionales son usar #ifndef, #define, #endif.

**Comment by sachondo7** [comment, marked as answer]
si, si puedes usar pragma once

**Reply by PabloNavarroY** [reply]
El setup que les pedimos que hagan es para que ocupen las mismas herramientas que vamos a ocupar nosotros para evaluarlos. En particular el compilador que ocupamos todos, gcc, soporta `#pragma once` oficialmente desde hace por lo menos 10 años.
